boilerplate/app/macversion.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
I think you should have a Python version check in here, especially if you're relying on `__future__`.

Reply:
`__future__` might even be able to be removed. All of the imports are a part of Python 3 and these files are only ever run via Python 3.